dashboard/run_data_collection.py

```diff
@@ -195,8 +195,8 @@ def collect_data(self, hpc: const.HPC, users: Iterable[str]):
 
         # user daily core hour usage
         user_ch_output = self.run_cmd(
-            "sreport -M {} -t Hours cluster AccountUtilizationByUser Users={} start={} end={} -n format=Cluster,Account,Login%30,Proper,Used".format(
-                hpc.value, " ".join(users), start_time, end_time
+            "sreport -M {} -t Hours cluster AccountUtilizationByUser Accounts={} Users={} start={} end={} -n format=Cluster,Account,Login%30,Proper,Used".format(
+                hpc.value, PROJECT_ID, " ".join(users), start_time, end_time
```

Comment:
1. make 'account/project_id a parsed arg for the whole function (because soon we will have 2 projects running)
2. if not supplied, use DEFAULT_ACCOUNT from qcore.const ( instead of a local variable)

Reply:
Done